hudi-common/src/main/java/org/apache/hudi/common/config/HoodieCommonConfig.java

```diff
@@ -78,18 +78,18 @@ public class HoodieCommonConfig extends HoodieConfig {
 
   public static final ConfigProperty<String> INCREMENTAL_READ_HANDLE_HOLLOW_COMMIT = ConfigProperty
       .key("hoodie.datasource.read.handle.hollow.commit")
-      .defaultValue(HollowCommitHandling.EXCEPTION.name())
+      .defaultValue(HollowCommitHandling.FAIL.name())
```

Comment:
thinking if we can name this 
"hoodie.datasource.read.concurrent.writes.plan"
or "hoodie.datasource.read.concurrent.writes.strategy"

Reply:
@nsivabalan is there a consensus on the name? i'll update it quickly then we can merge this.

Reply:
@xushiyan @nsivabalan How about `hoodie.read.timeline.holes.resolution.strategy`? This can apply to both sql and datasource, plus timeline can have holes irrespective if concurrent writes.

Reply:
lets go with "hoodie.read.timeline.holes.resolution.policy"

Reply:
Updated.